composer.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "owncloud/announcementcenter",
+  "config" : {
+    "platform": {
+      "php": "5.6.37"
+    }
+  },
+  "require": {
+  },
+  "require-dev": {
+    "phpunit/phpunit": "^5.7",
+    "jakub-onderka/php-parallel-lint": "^1.0",
```

Comment:
Are we keeping any lint?
@DeepDiver1975 @patrickjahns do we remove all evidence of PHP-lint-related testing?

Reply:
This has been sorted out today. We remove lint checks and add owncloud-coding-standard for PHP 5.6 - that gets us a PHP 5.6 syntax check.

Reply:
removed php-lint